src/earn/SafetyCard.tsx

```diff
@@ -16,8 +21,33 @@ const LEVEL_TO_MAX_HIGHLIGHTED_BAR: Record<Safety['level'], 1 | 2 | 3> = {
   high: 3,
 }
 
-export function SafetyCard({ safety }: { safety: Safety }) {
+function Risk({ risk }: { risk: SafetyRisk }) {
+  return (
+    <View style={styles.riskContainer} testID="SafetyCard/Risk">
+      <View style={styles.icon}>
+        {risk.isPositive ? (
+          <DataUp color={Colors.primary} testID="SafetyCard/RiskPositive" />
+        ) : (
+          <DataDown color={Colors.error} testID="SafetyCard/RiskNegative" />
+        )}
+      </View>
+      <View>
```

Comment:
Either here in a style to prevent text from long text from escaping the card bounds.

```suggestion
      <View style={{flexShrink: 1}}>
```

Reply:
ty, just went with flex: 1 so it can grow and shrink as needed